tests/.eslintignore

```diff
@@ -0,0 +1 @@
+fixtures/
```

Comment:
I tried to put .eslintignore in the tests folder in my branch and it didn't work, I had to add `tests/fixtures/` to the root .eslintrc

I'm wondering if this PR is passing only because you don't have any JS files that would otherwise fail eslint in the fixtures yet

Reply:
yeah, I think that's exactly right -- it seems eslint has a hard-coded ignore of prettier config files -- so I got lucky and this file didn't actually do anything.

I've moved the ignore entries to the directories where eslint and prettier are ran from (the package root)

updated.